server/prisma/schema.prisma

```diff
@@ -264,6 +264,7 @@ model users {
   updated_at       DateTime        @updatedAt
   id               Int             @id @default(autoincrement())
   name             String
+  image_url        String
```

Comment:
```suggestion
  image_url        String?
```
Having this be nullable better reflects the fact that the image_url is optional. It also means that `image_url: ''` can be deleted everywhere.